### ISSUE TITLE
Provide examples and structure for testing prospective APIs in REPL

### ISSUE THREAD

**Issue by kltm** [geneontology/noctua-repl#2]
We'd like to improve the ability to test and prototype APIs for minerva (mainly minerva-requests and bbop-response-barista) from a simple environment.

First: https://github.com/geneontology/noctua/issues/577 [DONE]

Note: https://github.com/geneontology/minerva/issues/200

**Comment by kltm** [comment]
tagging @goodb

**Comment by kltm** [comment]
@goodb 
The libraries and noctua-repl should be testable again.

Should I assume that you're running minerva and barista locally and that you want to be able to prototype commands to these local systems as JSON blobs?

**Comment by goodb** [comment]
Yes, running everything locally.  Want to test with commands as they would be issued by the libraries, so full requests (json and any request params)
